### PR TITLE
Conformance: Cover credentials tests

### PR DESCRIPTION
**Pull request by xpivarc**
### What this PR does
Cover SSH pub key propagation through
multiple propagation methods and password
propagation via GA.

Also don't make assert in helper and rather
return the error to be asserted at caller.

Fixes #

### Release note

```release-note
NONE
```

### CODE REVIEW

**Review by fossedihelm — APPROVED** [review]

File: tests/compute/credentials.go [hunk 1]
```diff
@@ -128,11 +128,11 @@ var _ = SIGDescribe("Guest Access Credentials", func() {
 			"fedora": []byte(customPassword),
 		}
 
-		It("[test_id:6221]should propagate user password", func() {
+		It("[test_id:6221]should propagate user password", decorators.Conformance, func() {
```
Comment (dankenigsberg) [inline tests/compute/credentials.go:131]: I'd vote against promoting this flavor of the test. It is too similar to test_id:6224 to merit a place in Conformance.

Comment (xpivarc) [reply tests/compute/credentials.go:131]: Second look, looks more like `6220`, agree to drop it

Comment (xpivarc) [reply tests/compute/credentials.go]: Done

**Pull request merged**